Increase curl timeout?

The Linux CI builds sometimes fail due to curl timeout, with error messages:
```
$ gmt grdimage @earth_relief_60m -JH10c -Baf -pdf map
Time spent = 5 seconds
gmt [ERROR]: Unable to download file https://oceania.generic-mapping-tools.org/gmt_hash_server.txt
gmt [ERROR]: Libcurl Error: Timeout was reached
gmt [ERROR]: GMT data server may be down - delay checking hash file for 24 hours
gmt [ERROR]: You can turn remote file download off by setting GMT_DATA_SERVER_LIMIT = 0.
gmt [ERROR]: Failed to get remote file https://oceania.generic-mapping-tools.org/gmt_hash_server.txt
gmt [ERROR]: Unable to obtain remote file @earth_relief_60m
grdimage [ERROR]: Error for input file: No such file (@earth_relief_60m.grd)
grdimage [ERROR]: Syntax error: Must specify one (or three) input file(s)
##[error]Bash exited with code '71'.
```

It's a little bit annoying. Maybe increase the curl timeout to 10 s?

--- a/src/gmt_remote.c
+++ b/src/gmt_remote.c
@@ -193,7 +193,7 @@ GMT_LOCAL size_t skip_large_files (struct GMT_CTRL *GMT, char* URL, size_t limit
 
 /* Deal with hash values of cache/data files */
 
-#define GMT_HASH_TIME_OUT 5L	/* Not waiting longer than this to time out on getting the hash file */
+#define GMT_HASH_TIME_OUT 10L	/* Not waiting longer than this to time out on getting the hash file */
 
 GMT_LOCAL int gmthash_get_url (struct GMT_CTRL *GMT, char *url, char *file, char *orig) {
 	int curl_err = 0;

